androidsdk/src/test/java/com/wootric/androidsdk/SurveyValidatorTest.java

```diff
@@ -63,6 +63,9 @@ public void checksEligibility_whenNotRecentlySurveyed() throws Exception {
     public void doesNotCheckEligibility_whenRecentlySurveyed() throws Exception {
         User user = testUser();
         EndUser endUser = testEndUser();
+        long createdAtJustNow = new Date().getTime();
```

Comment:
In fact, I do see comparisons on milliseconds, I don't see any conversions to seconds though...